apiary.apib

```diff
@@ -611,6 +611,7 @@ Get a list of companies.
                 + email: `info@piedpiper.eu` (string, required)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `Acme` (string, optional) - Filters on name, vat number, emails and telephones
+            + tag: `expo` (string, optional) - Filters on tag name
```

Comment:
Can we change this to `tags`? So that we can search on multiple tags at the same time? Kinda makes more sense to me :)

Reply:
We will make a follow up PR to handle this.